x-pack/plugins/index_lifecycle_management/public/application/sections/edit_policy/components/timeline/_timeline.scss

```
@@ -0,0 +1,77 @@
+$colorBarHeight: $euiSizeS;
+
+.ilmTimeline {
+  overflow: hidden;
+  width: 100%;
+
+  &__phasesContainer {
+    /*
+    * Let the delete icon sit on the same line as the phase color bars
+    */
+    display: inline-block;
+    width: 100%;
+
+    &__phase:first-child {
+      padding-left: 0;
+      padding-right: $euiSizeS;
+    }
+
+    &__phase:last-child {
+      padding-left: $euiSizeS;
+      padding-right: 0;
+    }
+
+    &__phase:only-child {
+      padding-left: 0;
+      padding-right: 0;
+    }
+
+    &__phase {
+      /*
+      * Let the phase color bars sit horizontally
+      */
+      display: inline-block;
+
+      padding-left: $euiSizeS;
+      padding-right: $euiSizeS;
+    }
+  }
+
+  &__deleteIconContainer {
+    /*
+    * Create a bit of space between the timeline and the delete icon
+    */
+    padding-left: $euiSizeM;
+  }
+
+  &__colorBar {
+    display: inline-block;
+    height: $colorBarHeight;
+    border-radius: calc(#{$colorBarHeight}/ 2);
+    width: 100%;
+  }
+
+  &__hotPhase {
+    width: var(--ilm-timeline-hot-phase-width);
+
+    &__colorBar {
+      background-color: $euiColorVis9_behindText;
```

It doesn't look like these are actually sitting underneath any text. So they should just use the plain vis colors for proper graphic style contrast.
```suggestion
      background-color: $euiColorVis9;
```